.devcontainer/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos/devtoolset-6-toolchain-centos7:latest
+
+LABEL \
+  description="Externpro Centos 7 Development Container" \
+  name="externpro-dev"
+
+USER 0
+RUN set -eux; \
+  yum --assumeyes --quiet update; \
+  yum install --assumeyes \
+    epel-release \
+    # This repository gives us more modern versions of some software, such as Git.
+    https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm; \
+  yum install --assumeyes \
+    cmake3 \
```

Comment:
which version of cmake3 does this install?

Reply:
3.17.3